mobile/mysterium/wireguard_connection_setup.go

```diff
@@ -164,19 +154,16 @@ func (c *wireguardConnection) Wait() error {
 }
 
 func (c *wireguardConnection) Stop() {
-	c.stateCh <- connection.Disconnecting
-	c.updateStatistics()
```

Comment:
This line was updating stats on disconnect. With this, you don't need to wait 5 seconds on e2e test to make sure we updated stats. And we are not losing the last part of stats.